PostPairsTrading.md

```diff
@@ -0,0 +1,313 @@
+In this heatmap, they exhibit a vibrant green colour, indicative of a high degree of cointegration, or, in simpler terms, a very low probability of not being cointegrated. They demonstrate low p-values suggesting their strength as candidates.
+
+ > 💡 As we can see, this pair of indexes is not the best candidate according to our ADF tests. However, we chose it because the tick data for their prices is publicly available. We used TickStory to obtain the data.
+
+![Prices](resources/cointegration.png)
+
+
+The plotted graph displays the prices of both indexes together, providing a clearer comparison that showcases the cointegration between them. The blue line represents SP500, and NASDAQ100 is represented by the green line. The close alignment of their price movements indicates that they are cointegrated to some extent. This means that, despite short-term deviations, the indices tend to move together as time goes on, maintaining a stable relationship. This graph was generated by [KX Dashboard](https://code.kx.com/dashboards/), which receives data from the Pairs Trading process and renders visualizations.
+
+Overall in this section we explained how the first part of our diagram works. We first discovered that, even in the randomness of the market, some assets show some similar movements. Then, based on that information and using the historical data from our tick architecture, we used the cointegration and ADF test to pinpoint a specific pair of them for our analysis: NASDAQ100 and SP500.
+
+![Arch 1st Part](resources/general-architecture-top.png)
+
+As mentioned earlier, the market is inherently random and doesn't always behave predictably. While NASDAQ100 and SP500 often follow similar trends, their individual values **can sometimes diverge significantly**. For instance, NASDAQ100 may rise while SP500 falls, or vice versa. 
+
+However, this presents **an opportunity for profit** because we know that these assets tend to revert to their shared mean over time. If one asset is **overpriced** and likely to decrease, we may consider **selling it** (going short). Conversely, if an asset is **underpriced** and expected to increase, we may consider **buying it** (going long). That is what we call Pairs Trading.
+
+> 💡 This strategy possesses financial characteristics: our **profitability remains unaffected by the broader market trends**, as our focus lies solely on the disparity between the two assets. It's about relative movements rather than absolute ones; we're indifferent to whether prices are rising or falling. This quality defines it as a **neutral market strategy**.
+
+## Determining how to calculate the spreads

```

Comment:
I want to discuss two aspects from this section with you. Let me know when you find the time! Just as a reminder for myself:
* HDB -> RPT link
* price_y and price_x source (rs?)